lib/matrix.gd

```diff
@@ -88,13 +97,22 @@ DeclareOperation("IsDiagonalMat",[IsMatrix]);
 ##  <Oper Name="IsUpperTriangularMat" Arg='mat'/>
 ##
 ##  <Description>
-##  returns true if mat has only zero entries below the main diagonal, false
-##  otherwise.
+##  return <K>true</K> if the matrix <A>mat</A> has only zero entries below
+##  the main diagonal, and <K>false</K> otherwise.
```

Comment:
Same gripe as above. Since it is not your "fault", I am not expecting you do anything about it... still, I'd find this text better:
> returns true if all entries of <mat> below the main diagonal are zero, and false otherwise.

And of course similarly for the `IsLowerTriangularMat` command.